sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/annotation/ReturnType.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.generic.core.annotation;
+
+/**
+ * Enumeration of return types used with {@link ServiceMethod} annotation to indicate if a method is expected to return
+ * a single item or a collection.
+ */
+public enum ReturnType {
+    /**
+     * Single value return type.
+     */
+    SINGLE,
+
+    /**
+     * Simple collection, enumeration, return type.
+     */
+    COLLECTION,
+
+    /**
+     * Long-running operation return type.
+     */
+    LONG_RUNNING_OPERATION
```

Comment:
We don't have long running operations in unbranded libraries. So, this should be removed.